packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -359,6 +361,11 @@ class TextMapPropagator {
       })
 
       this._extractBaggageItems(carrier, spanContext)
+
+      if (inGCPFunction()) {
```

Comment:
directly invoked google cloud functions (that weren't invoked by some datadog instrumented service) will have a `tracecontext` header. This header is injected by google for their own tracing solution. This causes the function to not show up as a Trace Root Span (as it's parent_id is being set to the id specified in the tracecontext header which google uses for their own tracing solution.

Because of this, we want to drop the span_id context for cloud functions so our cloud function root span shows up as trace root. Note: this makes trace propagation only possible through datadog context headers.